error.go

```diff
@@ -8,5 +15,108 @@ type ParseError struct {
 	Context *Context
 }
 
-// Cause returns the original cause of the error.
-func (p *ParseError) Cause() error { return p.error }
+// Unwrap returns the original cause of the error.
+func (p *ParseError) Unwrap() error { return p.error }
+
+// NOTE: The code bellow is taken from https://github.com/pkg/errors. BSD-2 license.
+
+// withStackError is the error that wraps caller stack.
+//
+// It is a helper type that formats stack trace properly.
+type withStackError struct {
```

Comment:
Just get rid of this, I think it's fine without it.

Reply:
Sure 🤓

Reply:
Done ✅